lib/reek/smells/attribute.rb

```diff
@@ -1,4 +1,6 @@
+require_relative 'smell_configuration'
```

Comment:
@chastell I agree.

This particular case is a bit odd since we're using constants from SmellConfiguration. I don't particularly like that system, but I also don't know yet how we can improve it.